frontend/src/Settings/CustomFormats/CustomFormats/ExportCustomFormatModalContentConnector.js

```diff
@@ -6,10 +6,10 @@ import { fetchCustomFormatSpecifications } from 'Store/Actions/settingsActions';
 import createProviderSettingsSelector from 'Store/Selectors/createProviderSettingsSelector';
 import ExportCustomFormatModalContent from './ExportCustomFormatModalContent';
 
-const blacklistedProperties = ['id', 'implementationName', 'infoLink'];
+const blocklistedProperties = ['id', 'implementationName', 'infoLink'];
```

Comment:
Omitted may be better here, similar to what you have in createHandleActions

Reply:
done